modules/azerionedgeRtdProvider.js

```diff
@@ -212,6 +212,7 @@ export const azerionedgeSubmodule = {
   name: SUBREAL_TIME_MODULE,
   init: init,
   getBidRequestData: getBidRequestData,
+  gvlid: IMPROVEDIGITAL_GVLID,
```

Comment:
Should we do a full rollback of all the consent code before performing these changes?

Reply:
The rollback will be part of SAS-15935 and it will be done manually, not with a reverse commit.